astroid/rebuilder.py

```diff
@@ -1855,7 +1929,9 @@ def visit_matchas(self, node: ast.MatchAs, parent: NodeNG) -> nodes.MatchAs:
             )
             return newnode
 
-        def visit_matchor(self, node: ast.MatchOr, parent: NodeNG) -> nodes.MatchOr:
+        def visit_matchor(
+            self, node: ast.MatchOr, parent: nodes.NodeNG
+        ) -> nodes.MatchOr:
             newnode = nodes.MatchOr(
                 lineno=node.lineno,
                 col_offset=node.col_offset,
```

Comment:
This file would deserve to be a package, github refuse to display the diff, like it's a ``package-lock.json``. (Just a passing observation, I'm not suggesting we immediately do something about it)